tests/unit_tests/test_dendrite.py

```diff
@@ -18,11 +18,22 @@
 # DEALINGS IN THE SOFTWARE.
 
 import pytest
+import typing
 import bittensor
 from unittest.mock import MagicMock, Mock, patch
 from tests.helpers import _get_mock_wallet
 
 
+class Dummy(bittensor.Synapse):
```

Comment:
if we want to add more test classes or share this one with other files, maybe we need to rename this class to `SynapseDummy`so we know what is that about. wdyt about that?

Reply:
Yes, good idea.